addons/stock/migrations/11.0.1.1/post-migration.py

```diff
@@ -185,9 +187,74 @@ def create_stock_move_line(env):
             LEFT JOIN stock_quant sq ON sq.id = smol.reserved_quant_id
             LEFT JOIN stock_production_lot spl ON spl.id = sq.lot_id
         WHERE sm.state = 'done'
-        GROUP BY sq.lot_id, spo.product_id""",
+        GROUP BY sq.lot_id, spo.product_id, spo.owner_id, spo.package_id,
+            spo.result_package_id""",
```

Comment:
I think you should also group by `sm.id`

Reply:
Or something similar, I dunno. I did a test, and I see that there is a need to create in some way more stock move lines with done moves.